src/app/liked/liked.component.scss

```diff
@@ -0,0 +1,14 @@
+.liked-btn {
+    border: 0;
+    width: 30px;
+    height: 30px;
+    border-radius: 50%;
+    background-color: var(--white-gray);
+    .label-icon {
+        width: 16px;
```

Comment:
add a default icon size with height 16 and width 16. use the same class when you can